gherkin-tests/src/main.rs

```diff
@@ -0,0 +1,107 @@
+use std::{cmp::Ordering, sync::Arc};
```

Comment:
We should move the gherkins stuff to its own file, something like `features.rs` and `features/given.rs`, `features/when.rs`, etc. WDYT?